erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -263,6 +264,94 @@
                         **_ERIC_GLOBAL_INITIALISATION_ERRORS, **_ERIC_GLOBAL_ERRORS, **_ERIC_TRANSFER_ERRORS,
                         **_ERIC_CRYPT_ERRORS, **_ERIC_IO_ERRORS, **_ERIC_PRINT_ERRORS}
 
+_ERIC_ERRORS_NO_RETRY = ["ALREADY_OPEN_UNLOCK_CODE_REQUEST", "ELSTER_REQUEST_ID_UNKNOWN", "ERIC_GLOBAL_PRUEF_FEHLER",
```

Comment:
One small thing: As this list is (only) intended for use outside of this file, the name shouldn't start with an underscore - that is convention for protected attributes/constants.

Reply:
Oh thanks! Changed :)